docs/spec_draft.md

```diff
@@ -420,6 +421,47 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.log
+
+`stablehlo.log(operand) -> result`
+
+### Semantics
+
+Performs element-wise logarithm operation on `operand` tensor and
+produces a `result` tensor, implementing the `log` operation from the IEEE-754
+specification. Numeric precision is implementation-defined.
+
+### Operands
+
+| Name | Type |
+|-|-|
+| `operand` | tensor of floating-point or complex element types |
+
+### Results
+
+| Name | Type |
+|-|-|
+| `result` | tensor of floating-point or complex element types |
+
+### Constraints
+
+  * Supported shapes: all static shapes.
+  * `result` must have the same type as that of `operand`.
```

Comment:
Let's also update constraints, similarly to https://github.com/openxla/stablehlo/pull/103:

```
### Constraints

  * (C1) `operand` and `result` have the same type.
```